### ISSUE TITLE
The issues stop updating

### ISSUE THREAD

**Issue by juvin-git** [headllines/github-daily#161]

**Comment by chucklu** [comment]
This scheduled workflow is disabled because there hasn't been activity in this repository for at least 60 days. Enable this workflow to resume scheduled runs. 
![image](https://user-images.githubusercontent.com/3004057/142848514-27b85c53-875c-41c7-897e-2e0b6afed588.png)

@timqian You might try this https://github.com/gautamkrishnar/keepalive-workflow to make sure the workflow will always keep alive.

**Comment by timqian** [comment]
The reason is the trending API we used to get the data can not be accessed and I can not find any stable API to do that. Any suggestion is welcomed

the current trending API we are using: https://github.com/huchenme/github-trending-api

**Comment by frandemona** [comment]
@timqian Seems someone in the issues found the fix: https://github.com/huchenme/github-trending-api/issues/154

> The reason is the trending API we used to get the data can not be accessed and I can not find any stable API to do that. Any suggestion is welcomed
> 
> the current trending API we are using: https://github.com/huchenme/github-trending-api

**Comment by chucklu** [comment]
@timqian  It stops working again since 26-Nov-2022.

**Comment by timqian** [comment]
> @timqian It stops working again since 26-Nov-2022.

Right, unfortunately the API we use is down. There seems no stable trending API. Any recommendation will be welcomed

**Comment by chucklu** [comment]
I have no idea about how this repository works, I just wonder what's the api concept do you mean?
Is this an official api? https://github.com/trending

**Comment by skipmaple** [comment]
I wrote a web crawler to fix it (crawl the top 10 projects on 12:00 everyday)

https://github.com/skipmaple/github-daily

**Comment by chucklu** [comment]
@skipmaple I have watched your repository months ago, but never receive a notification about the new issues.
The weird thing is that I am not listed on watchers of your repository https://github.com/skipmaple/github-daily/watchers
However, I definitely watching your repository

**Comment by chucklu** [comment]
@skipmaple my bad, I was supposed to watch your repository for all activity or custom with issues.  
I just switch to watch all activity of your repository, and I will check the notification in email tomorrow
